avatar.css

```diff
@@ -0,0 +1,25 @@
+.av-sm {
+  width: 3.5rem;
+  height: 3.5rem;
+}
+.av-md {
+  width: 7rem;
+  height: 7rem;
+}
+.av-lg {
+  width: 10rem;
+  height: 10rem;
+}
+.circle {
```

Comment:
You can shift this class to the utility folder so that it can be used in some other file also if required.

Reply:
OK will do it